umm-spec-lib/src/cmr/umm_spec/field_update.clj

```diff
@@ -6,11 +6,60 @@
   [cmr.umm-spec.date-util :as date-util]
   [cmr.umm-spec.umm-spec-core :as spec-core]))
 
+(def DataCenterURL
+  "Data center URL's content type."
+  "DataCenterURL")
+
+(def HomePage
+  "Data center URL's type."
+  "HOME PAGE")
+
 (defn field-update-functions
   "Partial update functions for handling specific update cases"
   [umm]
   {[:Instruments] (partial util/update-in-each umm [:Platforms])})
 
+(defn- get-home-page-url
+  "Get data center home page url from a list of related urls."
+  [related-urls]
+  (when (seq related-urls)
```

Comment:
Calling `first` on an empty list will return nil so the when line is unnecessary.

Reply:
Done.